packages/valory/skills/decision_maker_abci/behaviours/bet_placement.py

```diff
@@ -228,8 +231,12 @@ def async_act(self) -> Generator:
                 tx_submitter = self.matching_round.auto_round_id()
                 betting_tx_hex = yield from self._prepare_safe_tx()
 
-            payload = MultisigTxPayload(
-                agent, tx_submitter, betting_tx_hex, mocking_mode
+            payload = BetPlacementPayload(
+                agent,
+                tx_submitter,
+                betting_tx_hex,
+                mocking_mode,
+                wallet_balance=self.wallet_balance,
             )
```

Comment:
:warning: Here you disable the NONE event!! It cannot be triggered anymore!
Moreover, you are returning two different kinds of payloads in this behaviour!

Reply:
NONE event issue should be fixed. Also updated the type of payload.